scripts/malleus.sh

```diff
@@ -92,6 +97,7 @@ mvn \
 -Dorg.jitsi.malleus.room_name_prefix=$ROOM_NAME_PREFIX \
 -Dorg.jitsi.malleus.regions=$REGIONS \
 -Dremote.address=$HUB_URL \
+-DallowInsecureCerts=$ALLOW_INSECURE_CERTS \
```

Comment:
But wait

Reply:
I was testing with web.participant1.allowInsecureCerts=true ... web.participant2.allowInsecureCerts=true

Reply:
Have you tested it, maybe we need a global/default one ...